src/overlays/ovl_i4/D4BD0.c

```diff
@@ -154,7 +154,9 @@ void func_i4_8011B1E4(void) {
 
 #pragma GLOBAL_ASM("asm/us/rev0/nonmatchings/overlays/ovl_i4/D4BD0/func_i4_8011B380.s")
 
-#pragma GLOBAL_ASM("asm/us/rev0/nonmatchings/overlays/ovl_i4/D4BD0/func_i4_8011B3DC.s")
+Gfx* func_i4_8011B3DC(Gfx* arg0, unk_800E3A28* arg1) {
+    return arg0;
```

Comment:
Realise I probably missed this on the last review but general `Gfx* arg0` will be named as `Gfx* gfx`

```suggestion
Gfx* func_i4_8011B3DC(Gfx* gfx, unk_800E3A28* arg1) {
    return gfx;
```